test/e2e/performance/performance_test.go

```diff
@@ -145,15 +145,14 @@ var _ = Describe("performance test case", Serial, Label("performance"), func() {
 				defer cancel()
 				var curlCheckString string
 				if frame.Info.IpV6Enabled && !frame.Info.IpV4Enabled {
-					curlCheckString = fmt.Sprintf("curl -I -m 1 -g http://[%s]:80 --insecure", pod.Status.PodIP)
+					curlCheckString = fmt.Sprintf("curl -m 1 -g http://[%s]:80 --insecure", pod.Status.PodIP)
 				} else {
-					curlCheckString = fmt.Sprintf("curl -I -m 1 http://%s:80 --insecure", pod.Status.PodIP)
+					curlCheckString = fmt.Sprintf("curl -m 1 http://%s:80 --insecure", pod.Status.PodIP)
```

Comment:
It happened in previous ci that curl failed. However, we did not find any other abnormalities. We speculated that it might be due to network fluctuations, so we added a retry here.

curl --retry 10